viam-orb-slam3.go

```diff
@@ -57,9 +59,17 @@ const (
 	cameraValidationIntervalSec = 1.
 	parsePortMaxTimeoutSec      = 60
 	// time format for the slam service.
-	slamTimeFormat        = "2006-01-02T15:04:05.0000Z"
 	opTimeoutErrorMessage = "bad scan: OpTimeout"
 	localhost0            = "localhost:0"
+	AlgoName              = "orbslamv3"
+)
+
+// This gets created from the subAlgo parameter
+type OrbslamAlgorithm string
+
+const (
+	Mono OrbslamAlgorithm = "mono"
+	Rgbd OrbslamAlgorithm = "rgbd"
```

Comment:
[nit] I believe we decided to name that `subAlgo`, so I'd keep it consistent like this:

```suggestion
// SubAlgo defines various algorithms within orb-slam3
type SubAlgo string

const (
	Mono SubAlgo = "mono"
	Rgbd SubAlgo = "rgbd"
```

Reply:
Good idea. I didn't like `OrbslamAlgorithm`